Dockerfile

```diff
@@ -10,7 +10,7 @@ COPY --from=builder /go/src/github.com/openshift/cluster-node-tuning-operator/_o
 COPY --from=builder /go/src/github.com/openshift/cluster-node-tuning-operator/_output/gather-sysinfo /usr/bin/
 
 ENV ASSETS_DIR=/root/assets
-COPY assets $ASSETS_DIR
+COPY --from=builder /go/src/github.com/openshift/cluster-node-tuning-operator/assets $ASSETS_DIR
```

Comment:
@jmencak This is upstream Dockerfile only, right? Because the proper product builds use tuned from RPMs. Will this break downstream or not?

Reply:
@MarSik As far I understood from @jmencak we use Dockerfile.rhel9 for downstream, and Dockerfile is used for upstream only, but lets wait for a confirmation.

Reply:
> we use Dockerfile.rhel9 for downstream, and Dockerfile is used for upstream only

Starting with 4.16, this is only partially true.  From 4.16, [Dockerfile is also used by OKD](https://github.com/openshift/release/blob/4bacdfb86f76afc55a9d2c92b61fe57059eed1ab/ci-operator/config/openshift/cluster-node-tuning-operator/openshift-cluster-node-tuning-operator-release-4.16__okd-scos.yaml#L19).  So as long as we don't break OKD builds, AFAICS this PR is good to go.

/approve
/lgtm

/hold
for @jupierce to take a quick look so that this doesn't break something for OKD -- I'm familiar with the downstream OCP flow, but not so much with OKD build pipeline.

Note that the the patch causing the issue will go away anyway, [I have a cleanup PR for that](https://github.com/openshift/cluster-node-tuning-operator/pull/1151), reviews welcome /wink/ /wink/.